keyboards/mechlovin/infinity875/rev1/info.json

```
@@ -0,0 +1,30 @@
+{
+    "usb": {
+        "pid": "0x0875",
+        "device_version": "0.0.1"
+    },
+    "indicators": {
+        "caps_lock": "D6",
+        "scroll_lock": "B4"
+    },
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": false,
+        "rgblight": false,
+        "rgb_matrix": true
+    },
+    "rgb_matrix": {
+        "driver": "WS2812",
+        "max_brightness": 220,
+        "hue_steps": 10,
+        "sat_steps": 10,
+        "val_steps": 10,
+        "speed_steps": 15
+    },
```

Can maybe add `"cols": [null, null, null, ...]` to get `MATRIX_COLS` infered too, and remove it (and `MATRIX_ROWS`) from `config.h`
```suggestion
    },
    "matrix_pins": {
        "rows": ["D5", "D2", "D4", "D3", "D0", "D1"]
    }
```

---

For custom matrix, this should not go in info.json even if the define names match.